### ISSUE TITLE
Can't select file 

### ISSUE THREAD

**Issue by IAmDarush** [Angads25/android-filepicker#52]
In the files listview, clicking on a directory will open the directory, but clicking on a file does nothing unless you click on the checkbox next to file. This makes user experience very inconvenient.

Is there any work-around to this or could you please fix this?

**Comment by singhangadin** [comment]
It does, I added it [here](https://github.com/Angads25/android-filepicker/commit/f22da3899ecb703d4e85c715d2fa6c174146f38d)

**Comment by IAmDarush** [comment]
Thank you bro it was my bad. I was using  `compile 'com.github.angads25:filepicker:1.0.6'` version; I changed it to `com.github.angads25:filepicker:1.1.0`.

It works perfectly fine on **MULTI_MODE**, but when I use the **SINGLE_MODE** to select a single file, sometimes it does not select a file properly and I have to press it twice to be selected. If you can, go to a directory with multiple files, then select a file then select another file and then again select the previous file, you will see that it won't be select unless you try to press on it again.

**TL;DR**: using **SINGLE_MODE** one has to press a file twice to selected after being selected once. Could this be because of a Checkbox's behavior?

**Comment by singhangadin** [comment]
I think it's checkbox's behaviour that makes it odd. I have tried on my device and it is working fine.

**Comment by IAmDarush** [comment]
Thanks. I think if Checkbox was changed to **RadioButton** in **SINGLE_MODE** it would not give the problem.

**Comment by singhangadin** [comment]
I have been getting a lot of issues regarding odd behavior of checkbox upon clicks, Soon or later I will come up with a new alternative to it.

**Comment by singhangadin** [comment]
Hi, I have made few major changes for checkbox click issue. Will really appreciate if you could see to that and help me with bugs. Thank you.

**Comment by IAmDarush** [comment]
Thanks bro. So far very good. That is appreciable
